podcasts/Onboarding/Patron/PatronAppIconUnlock.swift

```diff
@@ -33,7 +38,7 @@ struct PatronAppIconUnlock: View {
         GeometryReader { proxy in
             ZStack {
                 if !isUnlocked {
-                    WelcomeConfetti(type: .normal)
+                    WelcomeConfetti(type: isPlus ? .plus : .normal)
```

Comment:
Any reason for the changes in this file?

Reply:
Yes, I'm stupid. Reverted!